nixos/modules/services/search/opensearch.nix

```diff
@@ -20,9 +23,9 @@ in
 {
 
   options.services.opensearch = {
-    enable = mkEnableOption (lib.mdDoc "Whether to enable OpenSearch.");
+    enable = mkEnableOption "OpenSearch";
```

Comment:
I think this broke the manual build.

Reply:
Oops, yeah, the `lib.mdDoc` still needs to be there.

Reply:
I will force push your commit :)

Reply:
Seems to have done the trick :)